easybuild/easyconfigs/g/GAMESS-US/compddi.patch

```diff
@@ -0,0 +1,20 @@
+The GAMESS-US build process requires that the MAXCPUS and MAXNODES values be
+set before compiling the DDI library. MAXNODES should be the number of nodes
+in your cluster, while MAXCPUS should be the number of CPU cores in the node
+with the most cores.
+
+Written by Benjamin Roberts, New Zealand eScience Infrastructure
+University of Auckland, Auckland, New Zealand
+--- gamess.orig/ddi/compddi	2013-05-04 08:14:15.000000000 +1200
++++ gamess/ddi/compddi	2015-01-23 15:37:42.510554228 +1300
+@@ -82,8 +82,8 @@
+ #     It is OK if these values are larger than your system,
+ #     but a serious error if they are smaller!
+ #
+-set MAXCPUS  = 16
+-set MAXNODES = 256
++set MAXCPUS  = 40
++set MAXNODES = 392
```

Comment:
so what if somebody's clusters have more nodes?

how about setting this ridiculously high? 10000000?

any idea how GAMESS-US uses this at all? looks quite useless, since they tell you that it's OK if these values are larger than what your system provides?